quarkus/deployment/src/main/java/org/keycloak/quarkus/deployment/CacheBuildSteps.java

```diff
@@ -67,12 +65,7 @@ void configureInfinispan(KeycloakRecorder recorder, BuildProducer<SyntheticBeanB
 
             try (BufferedReader reader = new BufferedReader(new InputStreamReader(url))) {
                 String config = reader.lines().collect(Collectors.joining("\n"));
```

Comment:
A follow-up task will be the possibility to parse the file in the build time and not just read it. It'll save another ~100ms.

Reply:
While the default configuration looks a build time configuration, it usually contains placeholders which are supposed to change at runtime. Please double-check with the store team if this is the way to go to avoid going into a direction which wouldn't work for Infinispan.